utils/build_preparation.py

```diff
@@ -135,22 +132,22 @@ def update(source_path, target_path):
     qualify_inputs(source_path, target_path)
     
     paths, files = get_source_items(REPLICATED_RESOURCES, source_path)
-    local_utilities.printf('source has  %d directories   and   %d files\n', len(paths), len(files))
+    printf('source has  %d directories   and   %d files\n', len(paths), len(files))
```

Comment:
Since this format ends with `\n`, could this line be replaced with:

```python
    print('source has %d directories   and   %d files', len(paths), len(files))
```

Reply:
I think you meant with string interpolation too